### DISCUSSION TITLE
coin daemon updates (BTC/LTC)

### DISCUSSION THREAD

**Discussion by cryptorex** [btcpayserver/btcpayserver#2540] [Technical Support] [answered]
Hello - I couldn't find any documentation on this.

Does maintenance update in the UI address updating the coin daemons?  

With taproot signaling and BTC core release to include activation height for taproot so was strategizing on the best way to address this.  Since running BTCPayServer I've not yet had to concern myself with any soft forks/forks.

Thanks!

**Comment by pavlenex** [comment, marked as answer]
We will explain this in our next release which s in around 3 weeks. Until then, you can follow along with the discussion [here](https://github.com/btcpayserver/btcpayserver-docker/pull/465)

But TLDR we will provide users with options to go both ways, if you want to go the Bitcon Core way, there's nothing you'd need to do, however, if you want to go with bip8-lottrue then you'd have to modify your environment variable. You can view at actual explanation in that PR [here](https://github.com/btcpayserver/btcpayserver-docker/pull/465/files#diff-1f5b443bc343315096e88846b6f43481077c489f88abee5fdb67829d8d76d828R23)

**Reply by cryptorex** [reply]
Excellent! thanks for the references and the quick reply.  

I will follow those links.
